app/bundles/AssetBundle/Event/RemoteAssetBrowseEvent.php

```diff
@@ -17,7 +17,7 @@ class RemoteAssetBrowseEvent extends CommonEvent
     /**
      * @var AbstractIntegration
      */
-    private $integration;
+    private \Mautic\PluginBundle\Integration\UnifiedIntegrationInterface $integration;
```

Comment:
Indeed, but it would probably import all the FQN names and make this PR much bigger.

If you agree, I can send the importing PR once the typed-related PRs are merged.

Reply:
At the moment its 505 files, so I'd recommend to handle it separately.

Reply:
Ok, thanks for checking.